src/Confluent.SchemaRegistry.Serdes.Protobuf/ProtobufUtils.cs

```diff
@@ -300,25 +300,25 @@ private static ISet<string> GetInlineTags(FieldDescriptorProto fd)
 
         public static FileDescriptorSet Parse(string schema, IDictionary<string, string> imports)
         {
-            IDictionary<string, string> allImports = new Dictionary<string, string>(BuiltIns);
-            imports?.ToList().ForEach(x => allImports.Add(x.Key, x.Value));
-            
-            var fds = new FileDescriptorSet();
-            fds.FileSystem = new ProtobufImports(allImports);
-            
-            fds.Add("__root.proto", true, new StringReader(schema));
-            foreach (KeyValuePair<string, string> import in allImports)
+            var allImports = imports != null
+                ? BuiltIns
+                    .Concat(imports)
+                    .GroupBy(s => s.Key) // remove duplicates, prefer built ins
+                    .ToDictionary(s => s.Key, s => s.First().Value)
+                : BuiltIns;
+            var fds = new FileDescriptorSet
             {
-                fds.AddImportPath(import.Key);
-                
-            }
+                FileSystem = new ProtobufImports(allImports)
+            };
+            fds.Add("__root.proto", true, new StringReader(schema));
+            fds.AddImportPath(""); // all imports are relative in the filesystem so must make import path just empty string
```

Comment:
This was the real bad bug that was breaking the functionality completely. Only one import path needed to be added. The empty import path since all paths in the file system are relative.